mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -0,0 +1,78 @@
+import {useState} from "react";
+import {renderSpan} from "src/component/editableText/renderSpan";
+import {Input} from "src/component/input/Input";
+import {KeyboardSymbols} from "src/utils/KeyboardSymbols";
+
+/**
+ * Cell item props
+ */
+interface EditableTextProps {
+
+  /**
+   * Cell item's text
+   */
+  text: string | number;
+
+  /**
+   * Function that update element on Enter click or unfocused
+   */
+  onChangeFinish: (value: string) => void;
```

Comment:
why does value = string? Shoud it be depend on type of EditableTextProps.text? shoul dit be generic?

Reply:
Change on generic